lib/ast.ex

```diff
@@ -35,12 +35,14 @@ defmodule UltraDark.AST do
   end
 
   def remap_with_gamma([computation | rest], new_ast \\ []) do
-    comp = remap_with_gamma(computation)
+    comp =
+      computation
+      |> remap_with_gamma
```

Comment:
@alexdovzhanyn if you do not have a more than one function do not do this, just call the function as you would without pipe.

Also, run `mix format` with latest elixir, you are missing the parenthesis.